export/cloud_datastore_admin.py

```diff
@@ -27,12 +27,14 @@
 import json
 import logging
 
+from core.platform import models
```

Comment:
I think this is going to run into problems for the same reason as #10800 -- this is a separate app, it doesn't know about core.

Reply:
Acknowledged.

Reply:
Now that we're moving to cloud functions, marking this thread resolved since these files are (tentatively) deleted by #10833.